lib/jenkins_pipeline/job.rb

```diff
@@ -31,7 +31,7 @@ def serialize
     private
 
     def build_status
-      return "unknown" if %w(running unknown failure).include?(@upstream_status)
+      return "unknown" if %w(running unknown failure aborted).include?(@upstream_status)
```

Comment:
Trying to assert only on the `success` status didn't work. Should we let it the way it is?